docs/scytl.rst

```diff
@@ -0,0 +1,40 @@
+Scytl
+=========
+
+********
+Overview
+********
+
+Scytl, or Clarity Elections, is a company that creates a tool for publishing election results in real-time. It's used in the U.S. by several states and over 800 counties, including Georgia, Colorado, Arkansas, and Dallas County, Texas.
+
+For each participating election administrator, they publish a site with results that can be published on election night. Unfortunately, while that site contains downloadable data, that data is formatted in a complex way, making it difficult for developers to fetch election results. In general, their results either come zipped in either an unformatted text file or a complex XML document. Summary results come as a zipped CSV, but just contain top-line results. The JSON that powers the site results is even more complicated.
+
+This connector provides methods to download the latest election results from their site and formats them into readable lists of dictionaries, which can easily be converted into a Parsons Table or Pandas dataframe.
+
+Because this connector is can be useful for live reporting, it also contains a polling feature. As long as the class is instantiated, it will only fetch results that are new since the previous fetch of that method. To skip this feature, set force_update to true on any of the fetch methods.
+
```

Comment:
The vast majority of our connectors (maybe all of them, other than this one?) require authentication. It seems like this doesn't because the data source is public? It would be useful to call that out so that a reader doesn't think that authentication is required but not documented. You can do something like
```

.. note::
  Authentication
    All endpoints for Scytl are public, and do not need authentication.
```